sudachipy/command_line.py

```diff
@@ -111,6 +113,20 @@ def _command_build(args, print_usage):
         builder.build(args.in_files, rf, wf)
 
 
+def _command_link(args, print_usage):
+    if args.unlink:
+        unlink_default_dict_package(msg=True)
```

Comment:
like `run` function in `command_line.py`, using `output` is consistent with current code.
we can control like
```
if PRINT_MESSAGE:
    unlink_default_dict_package(output=sys.stdout)
else:
    with open(os.devnull, 'w') as f:
        unlink_default_dict_package(output=f)
```
and simplify function removing if statements.

Not only this `run` function, `Tokenizer.tokenize` function follows this style.
So now I'd like you to follow this style.

(I haven't consider what strategy is the best for SudachiPy.)

Reply:
It's very smart! I'd like to use this strategy instead of msg flag.